Lib/timeit.py

```diff
@@ -17,6 +17,9 @@
   -p/--process: use time.process_time() (default is time.perf_counter())
   -v/--verbose: print raw timing results; repeat for more digits precision
   -u/--unit: set the output time unit (nsec, usec, msec, or sec)
+  -t/--target_time T: if number is 0 the code will run until it
+                      takes *at least* ``target_time`` seconds
+                      (default: 0.2)
```

Comment:
Let's use a dash in the CLI option.

Also, this might clarify we mean the number option, and not just the number used for target time.


```suggestion
  -t/--target-time T: if --number is 0 the code will run until it
                      takes *at least* ``target_time`` seconds
                      (default: 0.2)
```